README.md

```diff
@@ -1,20 +1,213 @@
-# AOEU React UI Component Library
+# AOEU UI Common
 
-This repository is a collection of shared React UI components for use in
-dependent projects (i.e. web and native app projects).
+This repository is a collection of shared UI resources for use in dependent projects (i.e. web and native app projects). It is configured and managed as a monorepo using [Lerna](https://lerna.js.org) and [Yarn Workspaces](https://classic.yarnpkg.com/en/docs/workspaces/).
 
 ## Dependencies
 
-* [React](https://reactjs.org/)
-* [Storybook](https://storybook.js.org)
+These are the main libraries and tools that are used in this project. They will all be installed and available for use when you've cloned the repository and run `yarn install`.
+
+* [React](https://reactjs.org/) - a JavaScript library for building component-based user interfaces
+* [Storybook](https://storybook.js.org) - a tool that facilitates developing, testing and reviewing the functionality of UI components in isolation.
+* [Lerna](https://lerna.js.org) - a tool for managing JavaScript projects with multiple packages that facilitates inter-package dependencies during development, package versioning and CHANGELOG generation, and publishing.
 
 ## Tasks
 
+* `yarn run eslint` - runs `eslint` for static analysis of JS modules and components
+* `yarn run mdlint` - runs `markdownlint` for status analysis of markdown documents
+* `yarn run lint` - runs both `eslint` and `mdlint` tasks
+* `yarn run test` - runs unit tests for JS modules and components with [Jest](https://jestjs.io/)
 * `yarn run storybook` - runs a [local instance](http://localhost:6006) of Storybook
+* `yarn run sb` - also runs Storybook (aliases the `storybook` task) (yes, I'm lazy)
 * `yarn run build-storybook` - builds a static instance of Storybook for deployment/hosting/reference
```

Comment:
Did notice a possible performance warning here. Something to keep in mind for the future.